src/main/java/com/iexec/common/web/ApiResponseBody.java

```diff
@@ -16,14 +16,18 @@
 
 package com.iexec.common.web;
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import java.util.List;
 
 @Data
 @Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
```suggestion
@AllArgsConstructor
@JsonInclude(Include.NON_NULL)
```
?

Reply:
What brings the JsonInclude annotation ?
Is it to avoid serializing properties with null ?

Reply:
I'll take a look on this

Reply:
See 97886ec